test/provision/rpm.sh

```diff
@@ -1,8 +1,8 @@
 #!/bin/bash -ex
 /vagrant/provision/generated/common.sh
 
-wget -q -O /etc/yum.repos.d/jenkins.repo @@RPM_URL@@/@@ARTIFACTNAME@@.repo
-rpm --import @@RPM_URL@@/jenkins-ci.org.key
+wget -q -O /etc/yum.repos.d/@@ARTIFACTNAME@@.repo @@RPM_URL@@/@@ARTIFACTNAME@@.repo
```

Comment:
The actual repo file does *not* use artifact names in the repo name, in docs for installation, even for enterprise distribution. 

This needs to be in a separate PR, because we're actually changing behavior here and documentation needs to change at same time.